<launch>
	<arg name="rss_name" 	default="/rss_robot"/>
	
	<!-- Manually run rss acquire because sudo -->
	<!--node pkg="rss" name="acquire" type="acquire"/-->

	<node pkg="wifi_nav"	name="wifi_node"	type="wifi_node">
		<remap from="/rss_avg"	to="$(arg rss_name)"/>	
	</node>

	<node pkg="wifi_nav"	name="wifi_nav"	type="rt1_nav">
		<param name="origin_x"    value="0"/>
		<param name="origin_y"	  value="0"/>
		<param name="bound_x_min" value="-100"/>
		<param name="bound_x_max" value="100"/>
		<param name="bound_y_min" value="-100"/>
		<param name="bound_y_max" value="100"/>
	</node>
</launch>
